lib/utils/database.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:path/path.dart';
+import 'package:sqflite/sqflite.dart';
+import 'dart:async';
+
+Future<Database> createDatabase() async {
+  final Future<Database> database = openDatabase(
+    join(await getDatabasesPath(), 'chirpnets.db'),
```

Comment:
Is this a local sqlite db on the device?

Reply:
Yessir

Reply:
Aight just making sure! :) 👍